package.json

```diff
@@ -27,6 +27,7 @@
     "build:operation": "(cd packages/airnode-operation && yarn run build)",
     "build:protocol": "(cd packages/airnode-protocol && yarn run build)",
     "build:validator": "(cd packages/airnode-validator && yarn run build)",
+    "build:utilities": "(cd packages/airnode-utilities && yarn run build)",
```

Comment:
I wonder if we want to use a separate pacakge for this 🤔

Reply:
We can discuss it further on the call later? I'm flexible.

Reply:
Yeah, essentially what exactly to put into this package and what do we want from it in the long term...